target/linux/bmips/dts/bcm6369-netgear-evg2000.dts

```diff
@@ -0,0 +1,242 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "bcm6368.dtsi"
+
+/ {
+	model = "Netgear EVG2000";
+	compatible = "netgear,evg2000", "brcm,bcm6369", "brcm,bcm6368";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_red;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 25 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 26 GPIO_ACTIVE_LOW>;
```

Comment:
And GPIO#26?

Reply:
It's also wrong in the bcm63xx target. 
https://github.com/openwrt/openwrt/blob/openwrt-23.05/target/linux/bcm63xx/dts/bcm6369-netgear-evg2000.dts#L36

Maybe the correct wps gpio button was 36?. Since I can't check it, I'll delete it.

Reply:
I think that this one is fine, because it's documented in https://openwrt.org/toh/netgear/evg2000?s[]=evg2000#buttons